modules/wsl-conf.nix

```
@@ -85,9 +85,10 @@ with lib; {
 
     environment.etc."wsl.conf".text = generators.toINI { } config.wsl.wslConf;
 
-    warnings = (optional (config.wsl.wslConf.boot.systemd && !config.wsl.nativeSystemd)
+    warnings = optional (config.wsl.wslConf.boot.systemd && !config.wsl.nativeSystemd)
       "systemd is enabled in wsl.conf, but wsl.nativeSystemd is not enabled. Unless you did this on purpos, this WILL make your system UNBOOTABLE!"
```

```suggestion
      "systemd is enabled in wsl.conf, but wsl.nativeSystemd is not enabled. Unless you did this on purpose, this WILL make your system UNBOOTABLE!"
```

When you're editing this file anyway, can you fix that typo as well?